forest-node/ansible/ansible.cfg

```diff
@@ -0,0 +1,8 @@
+[defaults]
+host_key_checking = false
```

Comment:
Yes, the host-checking might be cool but we want the easiest way possible for anybody to run this. In the case of this check, assuming a host is reinstalled and has a different key in ‘known_hosts’, there will be an error message until it is corrected (from experience). 

So If a new host is not in ‘known_hosts’, the control node may prompt for the confirmation of the key, which results in some form of manual/interactive intervention. So to avoid unevenness in the behaviors, then we disable host checking.

Reply:
Also, I must add that this increases speed of configurations with ansible

Reply:
Thanks for explaining. It's probably okay to leave it like this.